apps/cloud_federation_api/lib/Migration/Version1015Date202502262004.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * SPDX-FileCopyrightText: 2019 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace OCA\CloudFederationAPI\Migration;
+
+use Closure;
+use OCP\DB\ISchemaWrapper;
+use OCP\DB\Types;
+use OCP\Migration\IOutput;
+use OCP\Migration\SimpleMigrationStep;
+
+class Version1015Date202502262004 extends SimpleMigrationStep
+{
+
+	/**
+	 * @param IOutput $output
+	 * @param Closure $schemaClosure The `\Closure` returns a `ISchemaWrapper`
+	 * @param array $options
+	 * @return null|ISchemaWrapper
+	 */
+	public function changeSchema(IOutput $output, Closure $schemaClosure, array $options)
+	{
+		/** @var ISchemaWrapper $schema */
+
+
+		$schema = $schemaClosure();
+		$table_name = 'federated_invites';
+
+		if (! $schema->hasTable($table_name)) {
+
+			$table = $schema->createTable($table_name);
+			$table->addColumn('id', Types::BIGINT, [
+				'autoincrement' => true,
+				'notnull' => true,
+				'length' => 11,
+				'unsigned' => true,
+			]);
+
+			$table->addColumn('user_id', Types::STRING, [
+				'notnull' => false,
```

Comment:
Did you mix this up?
```suggestion
				'notnull' => true,
```

User ID can not be empty.